pkg/cmd/server/origin/rest/storage_options.go

```diff
@@ -0,0 +1,41 @@
+package rest
+
+import (
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	"k8s.io/kubernetes/pkg/genericapiserver"
+
+	configapi "github.com/openshift/origin/pkg/cmd/server/api"
+	quotaapi "github.com/openshift/origin/pkg/quota/api"
+	"github.com/openshift/origin/pkg/util/restoptions"
+)
+
+// StorageOptions returns the appropriate storage configuration for the origin rest APIs, including
+// overiddes.
+func StorageOptions(options configapi.MasterConfig) restoptions.Getter {
+	return restoptions.NewConfigGetter(
+		options,
+		&genericapiserver.ResourceConfig{},
+		map[unversioned.GroupResource]string{
+			{Resource: "clusterpolicies"}:       "authorization/cluster/policies",
```

Comment:
issue to inspect this when we start adding API groups.  We're going to need these to agree.